3rdparty/python/requirements.txt

```diff
@@ -30,7 +30,7 @@ pyopenssl==17.3.0
 pystache==0.5.3
 python-Levenshtein==0.12.0
 pywatchman==1.4.1
-PyYAML==5.1.2
+PyYAML==5.3.1
```

Comment:
How about we use `PyYAML>=5.3.1,<5.4`? Then we give slightly more flexibility for library users and still get a fairly tight pin for people using Pants as a binary.